### DISCUSSION TITLE
View unread comments

### DISCUSSION THREAD

**Discussion by romrell4** [community/community#110627] [Mobile] [answered]
When viewing a PR, it would be really nice to just see all the unresolved comments, so that I can know what more I need to resolve/comment on before merging. Right now it's all just cluttered (and collapsed) in the conversation page, so I have to go one by one through the reviews and expand the comments to see if/where my unresolved ones are.

![image](https://github.com/community/community/assets/15692420/57243e38-110d-4802-ab7e-359562344809)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.151.1 (124617884)
Device: iPhone 11 Pro (iOS 17.3.1)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @romrell4 thanks for writing in. I filed this feature request with the team!
